packages/react-core/src/components/Text/examples/Text.md

```diff
@@ -5,41 +5,52 @@ cssPrefix: pf-v5-c-content
 propComponents: ['TextContent', 'Text', 'TextList', 'TextListItem']
 ---
 
+The `Text` component provides simple, built-in styling for putting common blocks of HTML elements together. It establishes the block of content and styling within it for the elements listed in the `component` property(`h1` through `h6`, `p`, `a`, `small`, `blockquote` and `pre`), as well as the text component suite (`TextContent`, `TextList`, `TextListItem`).
```

Comment:
```suggestion
The `<Text>` component provides simple, built-in styling for putting common blocks of HTML elements together. It establishes the block of content and styling within it for the elements listed in the `component` property (`h1` through `h6`, `p`, `a`, `small`, `blockquote` and `pre`), as well as the text component suite (`TextContent`, `TextList`, `TextListItem`).
```

adding onto what Michael said - just some formatting fixes

Reply:
I like wrapping the component in `<>`, it helps differentiate a component from a prop. Tho should we also update this?

> (`TextContent`, `TextList`, `TextListItem`)

to 

> (`<TextContent>`, `<TextList>`, `<TextListItem>`).

Reply:
oh yes, missed those!